psen_scan_v2/InputPinID pin_id
string name
bool state
